azimuth/utils/project.py

```diff
@@ -51,6 +52,18 @@ def load_dataset_from_config(azimuth_config: AzimuthConfig) -> DatasetDict:
     return dataset_train_eval
 
 
+def save_config(azimuth_config: AzimuthConfig):
+    """Append config to configs.jsonl to retrieve past configs."""
+    with jsonlines.open(f"{azimuth_config.get_artifact_path()}/configs.jsonl", mode="a") as f:
+        f.write(azimuth_config.dict())
+
+
+def update_new_config(old_config: AzimuthConfig, change_set: Dict) -> AzimuthConfig:
+    new_config = old_config.copy(update=change_set, deep=True)
```

Comment:
Detail... I see where `change_set` comes from, but maybe we could get closer to `copy`'s syntax with `update`?
Also, is there a reason why the function is called `update_new_config` and not simply `update_config`?
```suggestion
def update_config(old_config: AzimuthConfig, update: Dict) -> AzimuthConfig:
    new_config = old_config.copy(update=update, deep=True)
```

Reply:
No, good idea! And `change_set` was already there, I agree that it wasn't a very good name.